### PR TITLE
1038 doc block pdf bug and pagination

### PR DESCRIPTION
**Pull request by nanetteCodes**

### CODE REVIEW

File: src/components/pagination/index.jsx [hunk 1]
```diff
@@ -46,7 +46,7 @@ const Pagination = ({
   }
 
   return (
-    <div className="ml-3">
+    <div className={`${className}`}>
```
[the remark this answers is not in the document]
Comment (nanetteCodes) [reply src/components/pagination/index.jsx]: className is a prop but yeah i removed the template literal and just did className={className}

[the remark this answers is not in the document]
Comment (efjacobson) [reply src/components/pagination/index.jsx]: oh yeah typo but you got it :]

File: src/plugins/blocksmith/components/document-block-modal/selector.js [hunk 2]
```diff
@@ -20,11 +20,14 @@ export default (state, { block }) => {
...
   const isDocumentAssetSearchFulfilled = status === STATUS_FULFILLED;
-
+  const currentPage = (request && request.get('page')) || 1;
+  const requestVal = request;
   return {
     documentAssetNodes,
     documentAssetSort,
     documentNode: block.has('node_ref') ? getNode(state, block.get('node_ref')) : null,
+    currentPage,
+    requestVal,
```
Comment (efjacobson) [inline src/plugins/blocksmith/components/document-block-modal/selector.js]: @nanetteCodes you can just pass request, dont need to put it into a different var

File: src/plugins/blocksmith/components/document-block-modal/selector.js [hunk 3]
```diff
@@ -20,11 +20,14 @@ export default (state, { block }) => {
   const documentAssetSort = (request && request.get('sort').getValue()) || SearchAssetsSort.CREATED_AT_DESC.getValue();
   const documentAssetNodes = (response && response.get('nodes')) || [];
   const isDocumentAssetSearchFulfilled = status === STATUS_FULFILLED;
-
+  const currentPage = (request && request.get('page')) || 1;
```
Comment (efjacobson) [inline src/plugins/blocksmith/components/document-block-modal/selector.js]: @nanetteCodes you can just pluck this out of the request in render

**Comment by nanetteCodes** [comment]
@efjacobson  ok i think i got it all now :)

File: src/plugins/blocksmith/components/document-block-modal/index.jsx [hunk 4]
```diff
@@ -325,6 +339,19 @@ class DocumentBlockModal extends React.Component {
             }
           </ScrollableContainer>
         </ModalBody>
+        {
+          activeStep === 0 && !!documentAssetNodes.length
+              && (
+                <Pagination
+                  className="justify-content-center d-flex mt-2"
+                  currentPage={request.get('page') || 1}
+                  key="pager"
```
Comment (efjacobson) [inline src/plugins/blocksmith/components/document-block-modal/index.jsx]: @nanetteCodes remove key, its unnecessary here

**Pull request merged**